java/runtime-native/src/main/java/org/ray/spi/impl/StateStoreProxyImpl.java

```diff
@@ -1,201 +0,0 @@
-package org.ray.spi.impl;
```

Comment:
It's better to do `git mv ../StateStoreProxyImpl.java ../BaseStateStoreProxyImpl.java`, instead of removing this file and creating a new file, so that git can still retain the history and github can compare these 2 files side by side.

Reply:
gotcha